SPECS/atop/atop.spec

```diff
@@ -1,41 +1,35 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
 %define _hardened_build 1
-
+Summary:        An advanced interactive monitor to view the load on system and process level
 Name:           atop
 Version:        2.6.0
 Release:        7%{?dist}
-Summary:        An advanced interactive monitor to view the load on system and process level
-
 License:        GPLv2+
-URL:            http://www.atoptool.nl
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.atoptool.nl
 Source0:        http://www.atoptool.nl/download/%{name}-%{version}.tar.gz
 Source1:        atop.d
-
 Patch0:         nvme_support.patch
 Patch1:         atop-sysconfig.patch
 Patch2:         atop-2.3.0-newer-gcc.patch
 Patch3:         9cb119713b5e6be43671fe1856fb4bd49ff91fa7.patch
-
 BuildRequires:  gcc
-BuildRequires:  zlib-devel
-BuildRequires:  ncurses-devel 
+BuildRequires:  make
+BuildRequires:  ncurses-devel
 BuildRequires:  systemd
-BuildRequires: make
+BuildRequires:  zlib-devel
+Requires(post): systemd
+Requires(postun): systemd
+Requires(preun): systemd
 %if 0%{?rhel} >= 8 || 0%{?fedora}
```

Comment:
Lets cleanup the distro specific macros :)
This would be a comment across both the two package specs.